### ISSUE TITLE
Incorrect Haddock Documentation 

### ISSUE THREAD

**Issue by TrueBoxGuy** [sdiehl/repline#33]
In line 89 of System.Console.Repline, not enough arguments have been applied to `evalRepl`: 

```hs
main = evalRepl (pure ">>> ") cmd options (Just ':') (Word completer) ini
```

**Comment by adamwespiser** [comment]
Thank you! I've added a fix in the referenced PR, and will merge it in the next few days.

**Comment by adamwespiser** [comment]
Fix with PR #34
